samples/disk.yaml

```diff
@@ -1,27 +1,29 @@
 apiVersion: openebs.io/v1alpha1
 kind: Disk
 metadata:
-  name: disk-3d8b1efad969208d6bf5971f2c34dd0c
+  name: <name of disk> #should be unique like disk-random-unique-name
```

Comment:
lets not make it a template, as the idea was we can quickly add one sample disk which has some random values. We can have comment explaining each field. So lets have a sample yaml with some default/random values and put comments explaining each field.

Reply:
I added one sample disk.